backend/EasySpeak.Core/EasySpeak.Core.DAL/Migrations/20230405093442_RemovedTagTable.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EasySpeak.Core.DAL.Migrations
+{
+    public partial class RemovedTagTable : Migration
```

Comment:
This is not good practice to drop and recreate tables for data updating
You need to use SQL and write script for data cleaning